src/ComposerEventHandler.php

```diff
@@ -165,12 +164,33 @@ public function onPostAutoloadDump(Event $event): void
 
     private function updateFile(string $source, string $destination): void
     {
-        // TODO: if update happened, do merge here
+        $distDestinationPath = dirname($destination) . '/' . self::DIST_DIRECTORY;
+        $distFilename = $distDestinationPath . '/' . basename($destination);
+
+        $fs = new Filesystem();
+
         if (!file_exists($destination)) {
-            $fs = new Filesystem();
+            // First install config
             $fs->ensureDirectoryExists(dirname($destination));
             $fs->copy($source, $destination);
+        } else {
+            // Update config
```

Comment:
We have updated packages in `$this->updates`. Does it worth checking it here to save file operations?

Reply:
What do you mean? In this point we update/install the configs.

Reply:
Yes, I mean we know exactly which packages were updated (`updates` property) but we compare and copy files for all packages, even ones that were not affected.